tests/typecheck/fields/test_base.yml

```diff
@@ -81,15 +81,19 @@
         MyModel(nulltext=None)
         MyModel().nulltext=None
         reveal_type(MyModel().nulltext)  # N: Revealed type is "Union[builtins.str, None]"
+    monkeypatch: true
     installed_apps:
         - myapp
     files:
         -   path: myapp/__init__.py
         -   path: myapp/models.py
             content: |
+                from typing import Optional, Union
                 from django.db import models
+                from django.db.models.expressions import Combinable
+
```

Comment:
Why are these new imports required ?

Reply:
Oh yeah good catch! These are not required anymore since now we are overloading `__new__` previously I had manually typehinted the `null=True` test cases. Removed them now.